SConstruct

```diff
@@ -147,6 +154,7 @@ opts.Add('vsproj', "Generate Visual Studio Project. (yes/no)", 'no')
 opts.Add('warnings', "Set the level of warnings emitted during compilation (extra/all/moderate/no)", 'no')
 opts.Add('progress', "Show a progress indicator during build (yes/no)", 'yes')
 opts.Add('dev', "If yes, alias for verbose=yes warnings=all", 'no')
+opts.Add('num_jobs', "Number of parallel builds", '2')
```

Comment:
That's a misleading option, as it's used only for vsproj. What it advertises is what the actual `-j` option does. Something like `vsproj_jobs` included below `vsproj` would be clearer.

Reply:
Roger, I will fix it.